source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -156,10 +159,31 @@ class StreamHandleWrapper : public Filters::Common::Lua::BaseLuaObject<StreamHan
    * @param 2 (table): A table of HTTP headers. :method, :path, and :authority must be defined.
    * @param 3 (string): Body. Can be nil.
    * @param 4 (int): Timeout in milliseconds for the call.
+   * @param 5 (bool): Flag. If true, filter continues without waiting for HTTP response from upstream service.
    * @return headers (table), body (string/nil)
    */
   DECLARE_LUA_FUNCTION(StreamHandleWrapper, luaHttpCall);
 
+  /**
+   * Perform an HTTP call to an upstream host.
+   * @param 1 (string): The name of the upstream cluster to call. This cluster must be configured.
+   * @param 2 (table): A table of HTTP headers. :method, :path, and :authority must be defined.
+   * @param 3 (string): Body. Can be nil.
+   * @param 4 (int): Timeout in milliseconds for the call.
+   * @return headers (table), body (string/nil)
+   */
+  DECLARE_LUA_FUNCTION(StreamHandleWrapper, luaHttpCallBlocking);
```

Comment:
is still necessary now that we're using the parameterized function instead? as in, they don't need to be LUA functions